integration_tests/sdk/aqueduct_tests/test_functions/simple/model.py

```diff
@@ -3,29 +3,28 @@
 
 @op()
 def dummy_sentiment_model(df):
-    df["positivity"] = 123
+    columns = df.columns
```

Comment:
These changes to the dummy models are so both pandas and Spark dataframes can use the same dummy models. I'll make the changes to the other test functions in another PR.